torch/distributed/distributed_c10d.py

```diff
@@ -3105,7 +3111,8 @@ def send_object_list(object_list, dst, group=None, device=None):
         device (``torch.device``, optional): If not None, the objects are
             serialized and converted to tensors which are moved to the
             ``device`` before sending. Default is ``None``.
-
+        group_dst (int, optional): Destination rank on ``group``.
```

Comment:
if you specify `group_dst` is there a requirement to also specify `group`?

Reply:
no. Technically, group defaults to 'default_group' and that's documented.  So I allowed using group_dst along with default group.  (This means in that case group_dst would be == global_dst).